srcpkgs/noisetorch/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'noisetorch'
+pkgname=noisetorch
+version=0.11.3
+revision=1
+wrksrc=NoiseTorch-${version}
+build_style=go
+go_ldflags="-X main.version=${version} -X main.distribution=voidlinux"
+go_import_path="."
+depends="polkit"
```

Comment:
Hello, upstream maintainer here. @ReneganRonin linked this PR upstream and i look the liberty to quickly glance over it.

Since you set `CAP_SYS_RESOURCE` in the package and the binary is managed by your package manager, polkit should not be a dependency. We only use `pkexec` from polkit if the binary manages itself. As long as the binary always has `CAP_SYS_RESOURCE` this should not be required.

Reply:
I see im going to remove that dependency then. thanks for the suggestions :D